src/applications/disability-benefits/disability-rating-calculator/components/CalculatedDisabilityRating.jsx

```diff
@@ -26,7 +26,7 @@ class CalculatedDisabilityRating extends React.Component {
           className="vads-u-font-weight--bold"
         >
           <div>Your VA disability rating</div>
-          <div className="vads-u-font-size--2xl vads-u-line-height--1">
+          <div className="combined-rating vads-u-font-size--2xl vads-u-line-height--1">
```

Comment:
Should we also wrap the `this.props.calculatedRating.rounded` value below in a selector so we can do an assertion on that?

Reply:
Replaced class with data-e2e="combine-rating" attribute.

The .expect... still works after above change -- did you mean we prefer assert over expect?

Reply:
Resolving this as the PR's been Approved.